contracts/UFragmentsPolicy.sol

```diff
@@ -29,6 +29,7 @@ contract UFragmentsPolicy is Ownable {
     using SafeMathInt for int256;
     using UInt256Lib for uint256;
 
+    /// @notice DEPRECATED.
     event LogRebase(
```

Comment:
We need to update and redeploy the subgraph index based on either `LogRebase` and `Rebase`.